### PR TITLE
Fix/sell button

### PR DESCRIPTION
**Pull request by eizegbu-bc**
## Description
Fixed Sell button
Added DefaultMessageType 
Added Capitalize Util

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/OrderSummary/template.success.tsx [hunk 1]
```diff
@@ -1,6 +1,11 @@
 import { Button, Icon, Text } from 'blockchain-info-components'
+import moment from 'moment'
+import React from 'react'
+
 import { BuyOrSell, displayFiat, getOrderDestination } from '../model'
+import { capitalize } from '../../../../utils/capitalize'
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/OrderSummary/template.success.tsx]: we have alias import for this. so just 
`import { capitalize } from 'utils/capitalize'`

File: packages/blockchain-wallet-v4-frontend/utils/capitalize.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export default function capitalize (string) {
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/utils/capitalize.js:1]: can you just add this function to `services/validationHelper.ts` it has related formatting things in there already

File: packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/OrderSummary/template.success.tsx [hunk 3]
```diff
@@ -12,11 +15,12 @@ import {
   getCounterCurrency,
   getOrderType
 } from 'data/components/simpleBuy/model'
+import styled from 'styled-components'
+
+import { BuyOrSell, displayFiat, getOrderDestination } from '../model'
+import { capitalize } from '../../../../utils/capitalize'
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/OrderSummary/template.success.tsx:21]: we have an alias for this import.  can just do `import { capitalize } from 'utils/capitalize'`. however lets move this function to `services/validationHelper` so your import will be `import { capitalizeFirstLetter } from 'services/validationHelper'

File: packages/blockchain-wallet-v4-frontend/utils/capitalize.js (hunk 2 above)
Comment (schnogz) [reply packages/blockchain-wallet-v4-frontend/utils/capitalize.js:1]: also rename it to `capitalizeFirstLetter`

**Review by schnogz — CHANGES_REQUESTED** [review]

File: packages/blockchain-wallet-v4-frontend/utils/capitalize.js (hunk 2 above)
Comment (eizegbu-bc) [reply packages/blockchain-wallet-v4-frontend/utils/capitalize.js:1]: Ok will do

**Review by schnogz — APPROVED** [review]

**Pull request merged**